content/docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -202,9 +183,13 @@ sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
 
 [admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#mutatingwebhookconfiguration-v1beta1-admissionregistration)
 configures when the webhook is invoked by Kubernetes. The default
-supplied with Istio selects pods in namespaces with label `istio-injection=enabled`.
-This can be changed by modifying the MutatingWebhookConfiguration in
-`install/kubernetes/istio-sidecar-injector-with-ca-bundle.yaml`.
+supplied with Istio selects pods in namespaces with label
+`istio-injection=enabled`.  The set of namespaces in which injection
+is applied can be changed by editing the MutatingWebhookConfiguration
+with `kubectl edit mutatingwebhookconfiguration
+istio-sidecar-injector`.
+
+> {{< warning_icon >}} The sidecar injector pod(s) should be restarted after modifying the mutatingwebhookconfiguration.
```

Comment:
`>` isn't needed here for a warning icon.

Reply:
https://istio.io/about/contribute/writing-a-new-topic/#adding-icons-emojis says both need to be included. And I didn't find any case where both weren't used in existing content. cc @geeknoid

Reply:
My bad @ayj.